src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -823,8 +867,79 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const { sovereignVenueId } = useSovereignVenueId({ venueId });
+
+  const { currentVenue: venue } = useConnectCurrentVenueNG(sovereignVenueId);
+
+  const [userStatuses, setUserStatuses] = useState<UserStatus[]>([]);
+
+  const {
+    isShown: hasUserStatuses,
+    show: showUserStatuses,
+    toggle: toggleUserStatus,
+  } = useShowHide(venue?.showUserStatus);
+
+  // Because this is not using the useForm validation. The use effect needs to manually open the dropdown with user statuses.
+  useEffect(() => {
+    if (!venue) return;
+    const venueUserStatuses = venue?.userStatuses ?? [];
+
+    setUserStatuses(venueUserStatuses);
+
+    if (venue.showUserStatus) {
+      showUserStatuses();
+    }
+  }, [showUserStatuses, venue]);
+
+  const removeUserStatus = (index: number) => {
+    const statuses = [...userStatuses];
+    statuses.splice(index, 1);
```

Comment:
think we can turn it into `const statuses = [...userStatuses].slice(index, 1);` unless there's a specific reason not to?

Reply:
Nope, both `splice` and `slice` have a return value of the removed elements, this won't work :/